cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -139,6 +139,8 @@ type AutoscalingOptions struct {
 	FilterOutSchedulablePodsUsesPacking bool
 	// IgnoredTaints is a list of taints to ignore when considering a node template for scheduling.
 	IgnoredTaints []string
+	// IgnoredLabels is a list of labels to ignore when comparing if two node groups are similar.
```

Comment:
State in comment that this is just an extension. That basic and cp-specific labels are always ignored.